src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicErrorFactsGenerator/Program.vb

```diff
@@ -27,7 +27,9 @@ Friend Module Program
         Dim infoCodeNames As New List(Of String)
         Dim hiddenCodeNames As New List(Of String)
         For Each line In From l In File.ReadAllLines(inputPath) Select l.Trim
-            If line.StartsWith("WRN_", StringComparison.OrdinalIgnoreCase) Then
+            If (line.Contains("_NextAvailable", StringComparison.OrdinalIgnoreCase)) Then
```

Comment:
>Contains [](http://example.com/codeflow?start=21&length=8)

EndsWith?

Reply:
No. The line ends with `= SomeNumber`, not this.